src/main/scala/org/deepdive/datastore/FileDataUtils.scala

```diff
@@ -17,4 +20,31 @@ object FileDataUtils extends Logging {
       reader.close()
     }
   }
+
+  def recursiveListFiles(dir: File): Array[File] = {
```

Comment:
Could you add unit tests for these extra functions?  The test coverage got decreased.